package.json

```diff
@@ -28,7 +28,8 @@
   "license": "MIT",
   "peerDependencies": {
     "react": "^16.8.1",
-    "react-native": "^0.61.3"
+    "react-native": "^0.61.3",
+    "react-native-gesture-handler": "^1.6.0"
```

Comment:
👀

Reply:
Is this really needed?

Reply:
Oh you know what, no. Sorry, somehow thought that TouchableOpacity was only from that package.

Reply:
Okay, fixed.